core/server/common/src/main/java/alluxio/master/PrimarySelectorClient.java

```diff
@@ -158,12 +212,14 @@ public void takeLeadership(CuratorFramework client) throws Exception {
         .forPath(mLeaderFolder + mName);
     LOG.info("{} is now the leader.", mName);
     try {
+      mLeaderZkSessionId = client.getZookeeperClient().getZooKeeper().getSessionId();
       waitForState(State.SECONDARY);
     } finally {
       LOG.warn("{} relinquishing leadership.", mName);
       LOG.info("The current leader is {}", mLeaderSelector.getLeader().getId());
```

Comment:
NIT: For all log messages, would it make sense to also add `mName` to messages for debugging?

Reply:
I'm not sure it makes sense to include `mName` anywhere except zookeeper path manipulation.